Command/BatchConsumerCommand.php

```diff
@@ -74,9 +78,15 @@ protected function execute(InputInterface $input, OutputInterface $output)
             define('AMQP_DEBUG', (bool) $input->getOption('debug'));
         }
 
+        $this->amount = (int)$input->getOption('messages');
+
+        if (0 > (int) $this->amount) {
+            throw new \InvalidArgumentException("The -m option should be null or greater than 0");
+        }
+
         $this->initConsumer($input);
 
-        return $this->consumer->consume();
+        return $this->consumer->consume((int)$this->amount);
```

Comment:
`$this->amount` already contains int (line 81)

Reply:
Fixed. Thank you!